src/test/java/ai/aliz/talendtestrunner/IntegrationTestRunner.java

```diff
@@ -26,8 +27,8 @@
 public class IntegrationTestRunner {
     
     private static final String API_URL = "apiUrl";
-    public static final String CONTEXT_PATH = "";
-    public static final String CONFIG_PATH = "";
+    public static final String CONTEXT_PATH = new File(IntegrationTestRunner.class.getClassLoader().getResource("contexts.json").getFile()).getPath();
```

Comment:
I meant to create properties file where similar to src/main/resources/application.properties
like a test.properties in the test resources folder ignored by git
It contains two property: context-path and test-config-path
And this properties file can be read by the test